test/tests/functional/pbs_Rrecord_resources_used.py

```diff
@@ -378,26 +351,21 @@ def test_t5(self):
 
         # Verify that the accounting logs have Resource_List.<resource> but no
         # R records.
-        m = self.server.accounting_match(
+        self.server.accounting_match(
             msg='.*Resource_List.*', id=jid1, regexp=True)
-        self.assertNotEqual(m, None)
-        m = self.server.accounting_match(
-            msg='.*R;' + jid1 + '.*resources_used.*', id=jid1, regexp=True)
-        self.assertIsNone(m)
-        m = self.server.accounting_match(
+        msg = '.*R;' + jid1 + '.*resources_used.*'
+        self.server.accounting_match(msg, id=jid1, regexp=True,
+                                     existence=False)
```

Comment:
In the previous call to accounting_match() and other calls in this file, you call accounting_match(msg=X).  In this case the message has been pulled out and set into a variable which is passed into accounting_match() as the first parameter.  Why the difference?